### PR TITLE
Add development nice-to-haves

### PR DESCRIPTION
**Pull request by michplunkett**
## Describe your changes
Added a static directory so you could edit CSS and JavaScript outside of the `base.html` file, updated the `CODEOWNERS` file to be valid with the current application structure, and added a CSS linter to help with front-end development.

## Checklist before requesting a review
- [x] pre-commit run --all-files (run before pushing)
- [x] pytest if applicable

```commandline
127.0.0.1 - - [29/Apr/2024 11:08:45] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 304 -
 * Detected change in '/Users/michaelp/Documents/GitHub/new-arrivals-chi/new_arrivals_chi/app/main.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 751-264-859
127.0.0.1 - - [29/Apr/2024 11:09:28] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [29/Apr/2024 11:09:28] "GET /static/css/main.css HTTP/1.1" 200 -
127.0.0.1 - - [29/Apr/2024 11:09:28] "GET /favicon.ico HTTP/1.1" 404 -
127.0.0.1 - - [29/Apr/2024 11:09:34] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [29/Apr/2024 11:09:34] "GET /static/css/main.css HTTP/1.1" 304 -
127.0.0.1 - - [29/Apr/2024 11:10:35] "GET /legal?lang=en HTTP/1.1" 200 -
127.0.0.1 - - [29/Apr/2024 11:10:35] "GET /static/css/main.css HTTP/1.1" 304 -
```

### CODE REVIEW

File: CODEOWNERS
Comment (michplunkett) [inline CODEOWNERS:10]: The previous CODEOWNERS setup wasn't working with the current architecture.

File: new_arrivals_chi/app/templates/base.html
Comment (michplunkett) [inline new_arrivals_chi/app/templates/base.html:26]: These are all now in the `main.css` file.

File: new_arrivals_chi/app/main.py
Comment (michplunkett) [inline new_arrivals_chi/app/main.py:26]: Added a static folder so that you could have CSS and JS files external to your HTML files.

File: new_arrivals_chi/app/static/css/main.css
Comment (michplunkett) [inline new_arrivals_chi/app/static/css/main.css:2]: It's the same CSS, I just sorted the properties.

**Review by sumslong — COMMENTED** [review]
Looks great to me on the organizational/frontend front, just want @aaronhaefner 's approval prior to merging as it as an update to pre-commit

**Review by sumslong — APPROVED** [review]

**Review by aaronhaefner — APPROVED** [review]

File: CODEOWNERS
Comment (aaronhaefner) [reply CODEOWNERS:10]: thanks for fixing

**Pull request merged**
